spec/services/asset_manager_service_spec.rb

```diff
@@ -5,15 +5,21 @@
 RSpec.describe AssetManagerService do
   describe "#upload_bytes" do
     it "uploads a byte stream to Asset Manager and returns the asset URL" do
-      asset = double(:asset, content_type: "type", filename: "foo/bar.jpg")
+      asset = double(:asset,
+                     content_type: "type",
+                     filename: "foo/bar.jpg",
+                     document: FactoryBot.build(:document))
```

Comment:
I think you can do `create(:document)` here and in the test below.

Reply:
Oh cool - I've updated. Note I went for build rather than create to avoid the db write.

Reply:
Ah. Ok.

Reply:
Yeah you often have to have it written to the db if you're going to do any finds or similar, but using build is way faster if you can do so.